runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/LazyJobBundleFactory.java

```diff
@@ -65,80 +68,67 @@
  * each client.
```

Comment:
This should be expanded. Why "Lazy" etc.

Reply:
Its lazy because it does not create the environment factory upfront when we create the JobBundleFactory instead it creates the EnvironmentFactory when it gets the first environment. Will update the comment.

Reply:
Makes sense. Is the qualifier needed at all then since this is really the only mode of operation we want.

Reply:
I agree, we can let go of the "Lazy" qualifier and instead document the behavior in the comment.